src/govuk/components/tag/_tag.scss

```diff
@@ -38,6 +42,21 @@
     }
   }
 
+  .govuk-tag--uppercase {
+    letter-spacing: 1px;
+    text-transform: uppercase;
+  }
+
+  .govuk-tag--lowercase {
```

Comment:
will this modifier be removed in v4.0.0?

Reply:
I guess so, this is why I'm debating whether or not this just needs to be a breaking change anyway?

Reply:
Unless of course we want this class for anyone that has their existing data in uppercase?